Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/FailedOrUnsupported.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    //this type is returned when all supported authorization types fail (403 error). This means either the provided credentials are invalid, or we do not support the authorization type of this url. 
+    //todo: try to determine the difference between wrong credentials and unsupported authorization types
```

Comment:
Technisch gesproken is dat verschil niet te vinden, niet in onze setup. Een goede webservice zal nooit reageren of een wachtwoord ongeldig is maar alleen dat de authenticatie niet gelukt is - maar de reden van de authenticatie faal is onbekend of dat credentials zijn of dat je de verkeerde methode gebruikt.

Reply:
Ik denk inderdaad dat de TODO weg kan